docs/example/public-hub-content.mdx

```diff
@@ -108,7 +108,7 @@ export SOURCE_REPO_REF="{ref}" # e.g. main
 export SOURCE_SSH_KEY="{key}"
 
 helm repo add capactio https://storage.googleapis.com/capactio-stable-charts
-helm upgrade capact --version {VERSION} capactio/capact -n capact-system --reuse-values --set hub-public.populator.enabled=true --set hub-public.populator.manifestsLocation.repository=${SOURCE_REPO_URL} --set hub-public.populator.manifestsLocation.branch=${SOURCE_REPO_REF} --set hub-public.populator.manifestsLocation.sshKey=${SOURCE_SSH_KEY}
+helm upgrade capact --version {VERSION} capactio/capact -n capact-system --reuse-values --set hub-public.populator.enabled=true --set hub-public.populator.manifestsLocations[0].repository=${SOURCE_REPO_URL} --set hub-public.populator.manifestsLocations[0].branch=${SOURCE_REPO_REF} --set hub-public.populator.manifestsLocations[0].sshKey=${SOURCE_SSH_KEY}
 ```
 
```

Comment:
It would be good to mention in this doc that we can use multiple manifest sources. I would suggest adding such note to every Tab:

```suggestion

> **NOTE**: You can provide multiple manifest sources by setting overrides using different indexes of the `hub-public.populator.manifestsLocations`.

```